src/components/PDFLink/Document/Standards.tsx

```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+import ReactPDF, { Document, Page, Text, View, StyleSheet, Image } from '@react-pdf/renderer';
+import { IStandards } from '../../../models/target';
+
+interface StandardsStyles {
+  flexRow: object;
+  flexColumnLeft: object;
+  flexColumnRight: object;
+  item: object;
+  desc: object;
+  code: object;
+}
+
+const styles: StandardsStyles = StyleSheet.create({
+  flexRow: {
+    flexDirection: 'row',
+    maxHeight: '100%'
+  },
+  flexColumnLeft: {
+    display: 'flex',
+    width: '25%',
+    padding: 5,
+    paddingRight: 8,
+    paddingTop: 10,
+    borderTop: '1px solid black',
+    borderRight: '2px solid black',
+    borderBottom: '1px solid black',
+    borderLeft: '2px solid black',
+    fontSize: 12,
+    textAlign: 'right'
+  },
+  flexColumnRight: {
+    display: 'flex',
+    flexDirection: 'column',
+    justifyContent: 'space-evenly',
+    width: '75%',
+    padding: 10,
+    paddingLeft: 8,
+    paddingTop: 10,
+    borderTop: '1px solid black',
+    borderBottom: '1px solid black',
+    borderRight: '2px solid black',
+    fontSize: 12
+  },
+  item: {
+    display: 'flex',
+    padding: '3px',
+    margin: 3
+  },
+  desc: {
+    margin: 5
+  }
+}) as StandardsStyles;
+
+export interface TargetProps {
+  content: IStandards[];
+}
+
+export const Standards = ({ content }: TargetProps) => {
+  return (
+    <View wrap style={styles.flexRow}>
+      <View style={styles.flexColumnLeft}>
+        <Text>Standards</Text>
+      </View>
+      <View style={styles.flexColumnRight}>
+        {content.map((element: IStandards) => {
+          const s = element.stdCode;
+          let array: string[];
+          array = s.split('.');
+
+          return (
+            <View wrap={false} style={styles.item} key={`${element.stdCode} - ${element.stdDesc}`}>
+              <Text key={`${array[1]} - ${element.stdDesc}`}>{`${array[4]}.${array[5]}.${
```

Comment:
Since you're relying on the order of values in the array, you should give each value a name so people understand what they are. You can accomplish this using variable array destructuring. You could so something like this:

```js
const [claim, target, grade] = array
```

More information here: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#Array_destructuring

Reply:
After looking at the code. I don't think this will clean this code up very much. I think we can use named variables to make it more readable but array restructuring only really works if we are going to use the elements of the array in order. However we don't use elements 0 or 3. I think this would possibly get more confusing.